src/styling.ts

```diff
@@ -61,3 +62,38 @@ browser.storage.onChanged.addListener((changes, areaname) => {
         retheme()
     }
 })
+
+export async function insertstyle(csscode: string) {
+    let tab = await webext.activeTab()
+    try {
+        //insert css to the current active tab
+        await webext.browserBg.tabs.insertCSS({ code: csscode })
```

Comment:
You don't need any of this code here. Please see my comments in excmds.ts : )

Reply:
ok.. that's something I should make a point. I don't know if this is important or not, but wouldn't it be more semantically concise if we call insertCSS from styling.ts than spreading it all over the code (in excmds or any other file)?

Reply:
No, I think insertCSS is simple enough that we can have it inline in excmds : )